kubernetes-maven-plugin/doc/src/main/asciidoc/inc/image/_example.adoc

```diff
@@ -1,16 +1,21 @@
-
-.Example
+.Example for <image>
 [source,xml,indent=0,subs="verbatim,quotes,attributes"]
 ----
 <configuration>
-  <!-- .... -->
```

Comment:
please do keep this as real XML comments, some syntax highlighting machines will break when processing this and also this code is not copy-pasteable since that is not a valid XML syntax.